content/guides/migrating.md

```diff
@@ -348,3 +348,51 @@ It is important to note that you will want to tell Babel to not parse these modu
   ]
 }
 ```
+
+Or if you're using the latest standards, use this method:
```

Comment:
This is already described somewhere above, see the chapter "Mixing ES2015 with AMD and CommonJS". Perhaps you could extend that chapter if there's info missing.